tests/context/test_case_context.py

```diff
@@ -0,0 +1,129 @@
+from response_operations_ui.contexts.case import build_case_context
+
+RU_REF = "49900000001"
+SURVEY_SHORT_NAME = "QBS"
+CASE_GROUP_ID = "6fef0397-f07b-4d65-8988-931cec23057f"
+CE_PERIOD = "1912"
+SURVEY_ID = "02b9c366-7397-42f7-942a-76dc5876d86d"
+
+
+def test_case_context_with_permissions(app, transitions_for_complete_case, expected_case_context_with_all_permissions):
```

Comment:
You might be able to use fixture params here

Reply:
Swapped for fixture params